### PR TITLE
STIJ-4: Add Mijn gent component

### PR DESCRIPTION
**Pull request by meirege**

### CODE REVIEW

File: components/31-molecules/building-blocks/mijn-gent-block/mijn-gent-block.twig [hunk 1]
```diff
@@ -0,0 +1,52 @@
+<div class="mijn-gent-block">
+  {% include '@link' with {
+    'link': 'javascript:(void);',
+    'modifier': 'toggle',
+    'text': '<i class="icon-user"></i> Profile'
```
Comment (Jeroen005) [inline components/31-molecules/building-blocks/mijn-gent-block/mijn-gent-block.twig]: Implementation when logged-in is missing.

File: components/31-molecules/building-blocks/mijn-gent-block/mijn-gent-block.twig [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+          ]
+        } %}
+      </section>
+
+    </div>
+    <footer>
+      {% include '@link' with {
+        'link': 'javascript:(void);',
+        'modifier': '',
+        'text': 'Uitloggen'
```
Comment (Jeroen005) [inline components/31-molecules/building-blocks/mijn-gent-block/mijn-gent-block.twig]: Uiloggen > Afmelden

File: components/31-molecules/building-blocks/mijn-gent-block/mijn-gent-block.twig (hunk 1 above)
Comment (Jeroen005) [reply components/31-molecules/building-blocks/mijn-gent-block/mijn-gent-block.twig]: Profile > Aanmelden

File: components/31-molecules/building-blocks/mijn-gent-block/mijn-gent-block.twig [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    <div class="toggle">
+      <div class="label">
+        {% include '@link' with {
+          'link': 'javascript:(void);',
+          'modifier': 'toggle',
+          'text': 'Username'
+        } %}
+      </div>
+      <div class="picture">
+        {% include '@image' with { 'src': '/styleguide/img/jpeg/avatar.jpeg', 'alt_text': 'Username' } %}
```
Comment (Jeroen005) [inline components/31-molecules/building-blocks/mijn-gent-block/mijn-gent-block.twig]: This image will be compiled and packaged in this location. Is it possible to move it to a directory thats excluded from our publish or use a external image?

File: components/31-molecules/building-blocks/mijn-gent-block/README.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+# Hamburger menu
```
Comment (Jeroen005) [inline components/31-molecules/building-blocks/mijn-gent-block/README.md]: Incorrect README.md

File: components/31-molecules/building-blocks/mijn-gent-block/_mijn-gent-block.scss [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+    width: 2.5rem;
+    height: 2.5rem;
+    margin-left: .625rem;
+
+    img {
+      border-radius: $radius-1;
+    }
+  }
+
+  .content {
```
Comment (Jeroen005) [inline components/31-molecules/building-blocks/mijn-gent-block/_mijn-gent-block.scss]: Can we position the dropdown on the login element itself?

Comment (meirege) [reply components/31-molecules/building-blocks/mijn-gent-block/_mijn-gent-block.scss]: It is positioned based on the toggle element (that is the username / icon / image). That's correct isn't it?

File: components/31-molecules/building-blocks/mijn-gent-block/README.md (hunk 4 above)
Comment (meirege) [reply components/31-molecules/building-blocks/mijn-gent-block/README.md]: Removed.

File: components/31-molecules/building-blocks/mijn-gent-block/mijn-gent-block.twig [hunk 6]
```diff
@@ -20,7 +18,7 @@
         } %}
       </div>
       <div class="picture">
-        {% include '@image' with { 'src': '/styleguide/img/jpeg/avatar.jpeg', 'alt_text': 'Username' } %}
+        {% include '@image' with { 'src': 'http://lorempixel.com/150/150/', 'alt_text': 'Username' } %}
```
Comment (Jeroen005) [inline components/31-molecules/building-blocks/mijn-gent-block/mijn-gent-block.twig]: https => https://lorempixel.com/150/150/people/

File: components/31-molecules/building-blocks/mijn-gent-block/_mijn-gent-block.scss (hunk 5 above)
Comment (Jeroen005) [reply components/31-molecules/building-blocks/mijn-gent-block/_mijn-gent-block.scss]: Yes it is, but `.mijn-gent-block` is a block so it's positioned with a width of 100%. If we change it to a `inline-block` the dropdown is always at the right on the toggle element.

Comment (meirege) [reply components/31-molecules/building-blocks/mijn-gent-block/_mijn-gent-block.scss]: True, changed it to inline-block and fixed the top and right values.

**Pull request merged**